samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinatorFactory.java

```diff
@@ -60,7 +60,7 @@ private ZkUtils getZkUtils(Config config, MetricsRegistry metricsRegistry) {
     return new ZkUtils(keyBuilder, zkClient, zkConfig.getZkConnectionTimeoutMs(), metricsRegistry);
   }
 
-  private String getJobCoordinationZkPath(Config config) {
+  public static String getJobCoordinationZkPath(Config config) {
```

Comment:
imo, this feels like a utility method and should be in its own class.
I am fine the way it is right now to unblock release